src/build/plugin-context.ts

```diff
@@ -14,6 +14,7 @@ import type { PrerenderManifest, RoutesManifest } from 'next/dist/build/index.js
 import type { MiddlewareManifest } from 'next/dist/build/webpack/plugins/middleware-plugin.js'
 import type { PagesManifest } from 'next/dist/build/webpack/plugins/pages-manifest-plugin.js'
 import type { NextConfigComplete } from 'next/dist/server/config-shared.js'
+import type { FunctionsConfigManifest } from 'next-with-cache-handler-v2/dist/build/index.js'
```

Comment:
No change needed - I continue to be a little worried about this way of getting types, but I still have no better solution to offer.

Reply:
Yeah, I don't have really answer for it other than possibly adoption of Adapters API once it become stable could stabilize types situation as this will be public API with proper semver considerations (tho even with Adapters API if we will look to support multiple Next.js major versions with same adapter, we likely would run in similar problem again)